front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/triggers/index.ts

```diff
@@ -122,16 +122,6 @@ async function handler(
     }
 
     case "DELETE": {
-      if (!agentConfiguration.canEdit) {
-        return apiError(req, res, {
-          status_code: 403,
-          api_error: {
-            type: "app_auth_error",
-            message: "Only editors can delete triggers for this agent.",
-          },
-        });
-      }
-
```

Comment:
this is ok to remove, we filter out on user's trigger below, you can delete only your personal triggers

Reply:
+ makes no sense as you can add triggers to dust while being a non editor